### DISCUSSION TITLE
Accurate Hitsounds Toggle

### DISCUSSION THREAD

**Discussion by EpiCreeper** [Quaver/Quaver#4793] [Feature Request]
Hitsounds are based on input, meaning there's always a slight delay between the press and sound. This carries over to autoplay because the hitsounds are based on the autoplay input and theres still delay. As a result of that delay, the hitsounds usually aren't synced with the song. That's usually not a problem because they are used to determine the accuracy of your input, but sometimes you are more interested in keeping the rhythm and knowing when you should press the note, and this kind of hitsounds isn't really helpful for that.

I want a new type of hitsounds to be added, one similar to the one in adofai, that would be completely accurate to the song and not your input. That kind of hitsounds would be useful for players that want to keep a steady rhythm or learn complicated timings. It would also be very useful for mappers to tell if their offset is correct.

**Comment by Emik03** [comment]
Just to be clear, by "new type of hitsound", you mean a toggle that a player is meant to use in the settings panel, and not the mapper putting specific hit/keysounds as accurate?

I'm definitely for the former, as playing on Linux tends to require high audio latency (in my case, only when I'm running many applications), which while the game does compensate within music thanks to an offset, it doesn't in hitsounds, making them unusable.

**Comment by EpiCreeper** [comment]
Oh yeah this is entirely player controlled. Making it baded on the chart would be a nightmare for everyone.
